compiler/src/dotty/tools/dotc/core/Definitions.scala

```diff
@@ -858,6 +858,7 @@ class Definitions {
   @tu lazy val TupleTypeRef: TypeRef = requiredClassRef("scala.Tuple")
   def TupleClass(using Context): ClassSymbol = TupleTypeRef.symbol.asClass
     @tu lazy val Tuple_cons: Symbol = TupleClass.requiredMethod("*:")
+  @tu lazy val EmptyTupleType: Symbol = ScalaPackageVal.requiredType("EmptyTuple")
```

Comment:
Seems wrong that `EmptyTupleType` and `EmptyTupleModule` are obtained by different means.

Reply:
I'm not sure I get the problem. Could you elaborate? This seems very similar to me to how `CanThrow` and the related 	`CanThrow` alias are obtained: https://github.com/lampepfl/dotty/blob/3.1.1/compiler/src/dotty/tools/dotc/core/Definitions.scala#L836-L837

If there is a better way, I suppose this can be changed and shouldn't be a fundamental issue.